easybuild/easyconfigs/d/dask/dask-0.12.0-intel-2016b-Python-2.7.12.eb

```diff
@@ -24,11 +23,4 @@ exts_list = [
     }),
 ]
 
```

Comment:
Exts_list lacks checksums here and some other places

Reply:
fixed in 21c110b5b5